zng/docs/spec.md

```diff
@@ -911,8 +911,7 @@ representing machine words are serialized in little-endian format.
 | `ip`       | 18 | 4 or 16  | 4 or 16 bytes of IP address                    | a string representing an IP address in [IPv4 or IPv6 format](https://tools.ietf.org/html/draft-main-ipaddr-text-rep-02#section-3) |
 | `net`      | 19 | 8 or 32  | 8 or 32 bytes of IP prefix and subnet mask     | a string in CIDR notation representing an IP address and prefix length as defined in RFC 4632 and RFC 4291. |
 | `type`     | 20 | variable | UTF-8 byte sequence of string representing type in the [TZNG type grammar](#42-type-grammar)  |  a UTF-8 string |
-| `error`    | 21 | variable | UTF-8 byte sequence of string of error message | a UTF-8 string |
-| `null`     | 22 |    0     | No value, always represents an undefined value | the literal value `-`                                 |
+| `null`     | 21 |    0    | No value, always represents an undefined value | the literal value `-`                                 |
```

Comment:
This doesn't match up with the `idReserved` or `idNull` entries in zng/types.go.

Reply:
I marked this as reserved for now.  Should we change "23" to "22" and get rid of the reserved slot? Would require one more rev to the sample-data repo.

Reply:
@mccanne: Revising the sample data is no biggie, so, whatever you think is cleanest long-term.